src/buildstream/_frontend/app.py

```diff
@@ -711,7 +711,7 @@ def _handle_failure(self, element, task, failure):
                             _Scope.BUILD,
                             self.shell_prompt,
                             isolate=True,
-                            usebuildtree="always",
+                            usebuildtree=True,
```

Comment:
How does this change relate?

Reply:
This is just a side effect of changing the various `Stream()` APIs (using the new type annotations for the changed APIs) and reviewing all of the call sites for these changed `Stream()` APIs.

There is no such thing as `"always"` as an answer to the question *"Shall I use a build tree for this shell"*, this does not technically fix any bug because `"always"` is a truthy value, but the signature takes a `bool`, so we now give it a `bool`.